apps/module-scan/src/scanners/plustek.test.ts

```diff
@@ -236,12 +240,13 @@ test('withReconnect', async () => {
   expect(getClient).toHaveBeenCalledTimes(4)
 
   // interacting with the good client works
-  await client.simulateLoadSheet(['/tmp/a.jpg', '/tmp/b.jpg'])
```

Comment:
This is an instance of where the new rule `no-floating-results` actually caught a problem.